src/OmniSharp.Abstractions/Utilities/Platform.cs

```diff
@@ -88,8 +88,8 @@ private static Platform GetCurrentPlatform()
                 }
 
                 var values = output.Split(' ');
-                var osName = values[0];
-                var osArch = values[1];
+                var osName = values[0].Trim();
+                var osArch = values[1].Trim();
```

Comment:
Will the array length ever possibly be `1`?

Reply:
Yes, if `uname` fails. Although, that's already a potential problem here and I haven't seen anything fail yet due to that.